moduleroot/spec/acceptance/nodesets/docker/debian-8.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I'm trying to disable the getty's here because they reply my system getty's. It's an issue with docker --privileged.